packages/explorer/test/integration/rest.integration.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/explorer
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+import {createClientForHandler} from '@loopback/testlab';
+import {RestServerConfig, RestComponent, RestServer} from '@loopback/rest';
+import {Application} from '@loopback/core';
+import {ExplorerComponent, ExplorerBindings} from '../..';
+
+describe('API Explorer for REST Server', () => {
+  let server: RestServer;
+
+  before(async () => {
+    server = await givenAServer({
+      rest: {},
+    });
+
+    // FIXME: How do we configure the API Explorer UI?
+    server.bind(ExplorerBindings.CONFIG).to({});
+
+    // FIXME: Should the ExplorerComponent contribute a handler or route?
+    const handler = await server.get(ExplorerBindings.HANDLER);
+
+    // FIXME: How to register routes to the rest server by phase or order
+    // FIXME: Should REST server automatically mounts the route based on registration
+    // via bindings such as `rest.server.routes`?
+    server.staticRouter.use('/explorer', handler);
```

Comment:
I find this very hacky.

Just the fact that `staticRouter` is a public property is alarming to me, as it opens door to all kinds of hacks that people can start making with our internal implementation details of `app.static()`. Let's fix that please and make `server.staticRouter` a private property.

Secondly, I thought the entire purpose of `app.static` was to allow the explorer component to serve swagger-ui static assets. I don't fully understand why you don't want to use it?

IMO, we should focus on `RestApplication` scenario in the first iteration because multi-server apps are not officially supported yet, and call `app.static()` in component constructor.

Thoughts?